controller/konnect/ops/ops_controlplane.go

```diff
@@ -145,17 +146,28 @@ func setGroupMembers(
 			)
 			if err := cl.Get(ctx, nn, &memberCP); err != nil {
 				return sdkkonnectcomp.Members{},
-					fmt.Errorf("failed to get control plane group member %s: %w", member.Name, err)
+					GetControlPlaneGroupMemberFailedError{
+						MemberName: memberCP.Name,
+						Err:        err,
+					}
 			}
 			if memberCP.GetKonnectID() == "" {
 				return sdkkonnectcomp.Members{},
-					fmt.Errorf("control plane group %s member %s has no Konnect ID", cp.Name, member.Name)
+					ControlPlaneGroupMemberNoKonnectIDError{
+						GroupName:  cp.Name,
+						MemberName: memberCP.Name,
+					}
 			}
 			return sdkkonnectcomp.Members{
 				ID: lo.ToPtr(memberCP.GetKonnectID()),
 			}, nil
 		})
 	if err != nil {
+		SetControlPlaneGroupMembersReferenceResolvedConditionFalse(
+			cp,
+			"SomeMemberNotResolved",
```

Comment:
Can we put this into a named `const` with a comment?

Reply:
Done.